### ISSUE TITLE
[low priority] Style the Editor pane

### ISSUE THREAD

**Issue by okjuan** [Wubbadub/ClubHub#76]
__Description:__
Make our Editor pane look pretty. For example, use colour to make hierarchy nicer to look at. Another idea is to make toggle the pane from solid --> see-through when the user moves the mouse off of it, and vice versa.

__Requires:__
* styling in editor.less

__Breakdown:__
- [ ] possibly tweak attributes in component files
- [ ] add some colours and stuff
- [x] play with see-through toggle idea

__Time Estimation:__
1-2 hrs.

**Comment by hwkr** [comment]
Tried solid see-through, it was painful.

**Comment by kelseyll** [comment]
I think we shouldnt  be worried about making our editor pretty until its fully functional. Can we worry about this until after the presentation today @brynn?

- We still can't even change the photo
- It doesn't indicate which element you are working on with a blue outline like in the mock-up.
- we can't remove items we don't want (what if a club doesn't have an Instagram? )

Tldr Im a bit worried about this being worked on right now since the editor needs sooo much work functionality wise. And our presentation luls.

edit: i need 2 calm down

**Comment by hwkr** [comment]
Yeah, for sure. It's marked Milestone 3 😛

**Comment by bndnio** [comment]
duplicate of #112
